recipes/r-bvar/meta.yaml

```
@@ -0,0 +1,79 @@
+{% set version = '1.0.4' %}
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-bvar
+  version: {{ version|replace("-", "_") }}
+
+source:
+  url:
+    - {{ cran_mirror }}/src/contrib/BVAR_{{ version }}.tar.gz
+    - {{ cran_mirror }}/src/contrib/Archive/BVAR/BVAR_{{ version }}.tar.gz
+  sha256: 440ce37924187183dd54e7e0f6ebf8e5faf52b1fbaac6914cca3f0c6bcc56b89
+
+build:
+  merge_build_host: True  # [win]
+  number: 0
+  noarch: generic
+  rpaths:
+    - lib/R/lib/
+    - lib/
+
+requirements:
+  build:
+    - {{ posix }}zip               # [win]
+    - cross-r-base {{ r_base }}    # [build_platform != target_platform]
+  host:
+    - r-base
+    - r-mvtnorm
+  run:
+    - r-base
+    - r-mvtnorm
+
+test:
+  commands:
+    - $R -e "library('BVAR')"           # [not win]
+    - "\"%R%\" -e \"library('BVAR')\""  # [win]
+
+about:
+  home: https://github.com/nk027/bvar
+  license: GPL-2.0-or-later
```

DESCRIPTION shows `License: GPL-3 | file LICENSE`, which implies:

```suggestion
  license: GPL-3.0-only
```